src/features/classification/Classification.scss

```diff
@@ -22,8 +22,4 @@
 .bdl-Classification-appliedByAiDetails {
     margin-bottom: var(--space-5);
     margin-top: var(--space-05);
-
-    &.bdl-AppliedByAiClassificationReason {
```

Comment:
removed specific override and just use default card padding, confirmed with PM & Design on changes